dwave/cloud/client/base.py

```diff
@@ -224,6 +224,10 @@ class Client(object):
         leap_api_endpoint (str, optional):
             Leap API endpoint.
 
+        leap_client_id (str, optional):
+            Leap OAuth 2.0 Ocean client id. Reserved for testing, otherwise
```

Comment:
Maybe should be named `_leap_client_id` and not documented for now?

Reply:
This is a config file option users will not care about, but it'll still be there. Unfortunately, `Client` class docs are currently the only place we document config options, and we should change that, especially with all the recent config changes.